core/templates/components/forms/custom-forms-directives/apply-validation.directive.ts

```diff
@@ -67,7 +67,8 @@ angular.module('oppia').directive('applyValidation', [
 
             var customValidator = function(viewValue) {
               ctrl.$setValidity(
-                frontendName, $filter(frontendName)(viewValue,
+                frontendName, $filter(frontendName)(
+                  viewValue,
                   filterArgs));
```

Comment:
Please move such lines above. I left this comment before too, please take a thorough look over all your changes.